tools/src/main/java/org/apache/kafka/trogdor/workload/ProduceBenchWorker.java

```diff
@@ -84,15 +90,32 @@ public void start(Platform platform, WorkerStatusTracker status,
             throw new IllegalStateException("ProducerBenchWorker is already running.");
         }
         log.info("{}: Activating ProduceBenchWorker with {}", id, spec);
-        // Create an executor with 2 threads.  We need the second thread so
-        // that the StatusUpdater can run in parallel with SendRecords.
-        this.executor = Executors.newScheduledThreadPool(2,
+        // Create an executor with 3 threads.  We need the second thread so
+        // that the StatusUpdater can run in parallel with SendRecords, and
+        // the third thread is for refreshing the partition info, if desired.
+        this.executor = Executors.newScheduledThreadPool(3,
```

Comment:
Should we bump this to 3 only if partition refresh is enabled?

Reply:
Done.